pages/form/Question4.jsx

```diff
@@ -1,34 +1,100 @@
 import { useState } from "react";
-import { Heading, Text, Textarea, Grid } from "@chakra-ui/react";
+
+import {
+  Heading,
+  Box,
+  Text,
+  Flex,
+  Select,
+  Stack,
+  Center,
+} from "@chakra-ui/react";
+import { modesOfTransport } from "../../utils/constants";
 import Layout from "../../components/Layout/Layout";
 import useForm from "../../components/FormProvider";
-import { BackButton, ContinueButton } from "../../components/LinkButton/LinkButton";
+import {
+  ContinueButton,
+  BackButton,
+} from "../../components/LinkButton/LinkButton";
 import Q4Progress from "../../public/images/progress-bar/q4-progress-dots.svg";
+import Q4Cloud from "../../public/images/clouds/cloud-q4.svg";
 
 export default function Question4() {
   const { answers, setAnswers } = useForm();
-  const [incentive, setIncentive] = useState(answers.incentive);
 
-  const saveAnswers = () => setAnswers( prev => ({...prev, incentive}) );
+  // selectedMode
+  const [selectedMode, setSelectedMode] = useState(
+    answers.mainTransportMode || ""
+  );
+
+  const saveAnswers = () =>
+    setAnswers((prev) => ({ ...prev, mainTransportMode: selectedMode }));
 
   return (
     <Layout isText={true} Progress={Q4Progress}>
-      <Heading as="h1" size="md" mt={6}>
-        What can council do to support and incentivise staff to choose more sustainable modes of transport to and from work?
-      </Heading>
-      <Text mt={4} w="100%">
-        For example: public transport subsidies or carpooling arrangements
-      </Text>
-      <Textarea
-        mt={8}
-        value={incentive}
-        onChange={(e) => setIncentive(e.target.value)}
-        placeholder="Suggest an incentive"
-      />
-      <Grid templateColumns="repeat(2, 1fr)" gap={4}>
+      <Box pos="absolute" top="10px" left="-90px" className="backbtn">
         <BackButton href="/form/Question3" onClick={saveAnswers} />
-        <ContinueButton href="/form/Question5" onClick={saveAnswers} />
-      </Grid>
+      </Box>
+
+      <Box margin={"50px 0px 50px"}>
+        <Q4Cloud />
+      </Box>
+
+      <Heading
+        fontSize="40px"
+        mt="6"
+        fontWeight={700}
+        maxWidth={"624px"}
+        textAlign={"center"}
+        marginBottom={"30px"}
+      >
+        Select the main way you travel to work
+      </Heading>
+
+      {/* Responsive buttons */}
+      <Flex maxWidth="full" margin={"60px 0px 50px"} flexDirection="column">
+        <Box fontSize="20px" mr="10px" textAlign="left">
+          <Text my={2} maxWidth={"550px"}>
+            For example, if you drive 2km to the train and then catch the train
+            for 15km, choose 'Train' as your way of travel.
```

Comment:
@Amutha37 , you need to be mindful of escaping the apostrophes in your text!

```suggestion
            for 15km, choose &apos;Train&apos; as your way of travel.
```